Simplistic/index.js

```diff
@@ -0,0 +1,185 @@
+	rank.style.textShadow = shadow;
+}
+socket.onopen = () => console.log("Successfully Connected");
+socket.onclose = event => {
+	console.log("Socket Closed Connection: ", event);
+	socket.send("Client Closed!");
+};
+socket.onerror = error => console.log("Socket Error: ", error);
+let pp = new CountUp('pp', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: " "
+	, decimal: "."
+});
+let fc = new CountUp('ppfc', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: " "
+	, decimal: "."
+});
+let h100 = new CountUp('h100', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: ""
+	, decimal: "."
+});
+let h50 = new CountUp('h50', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: ""
+	, decimal: "."
+});
+let h0 = new CountUp('h0', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: ""
+	, decimal: "."
+});
+let ss = new CountUp('ss', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: " "
+	, decimal: "."
+});
+let sb = new CountUp('sb', 0, 0, 0, .5, {
+	useEasing: true
+	, useGrouping: true
+	, separator: " "
+	, decimal: "."
+});
+socket.onmessage = event => {
+	try {
+		let data = JSON.parse(event.data)
+			, menu = data.menu
+			, play = data.gameplay
+			, hitGrade = data.gameplay.hits.grade.current
+			, hdfl = (data.menu.mods.str.includes("HD") || data.menu.mods.str.includes("FL") ? true : false)
+			, tempGrade = ""
+			, tempColor = ""
+			, tempShadow = "";
+		// Rank Check
+		function rankCheck() {
+			switch (hitGrade) {
+			case "SS":
+				tempGrade = hitGrade;
+				tempColor = (hdfl ? "#e0e0e0" : "#d6c253");
+				tempShadow = (hdfl ? "0 0 0.5rem #e0e0e0" : "0 0 0.5rem #d6c253");
+				break;
+			case "S":
+				tempGrade = hitGrade;
+				tempColor = (hdfl ? "#e0e0e0" : "#d6c253");
+				tempShadow = (hdfl ? "0 0 0.5rem #e0e0e0" : "0 0 0.5rem #d6c253");
+				break;
+			case "A":
+				tempGrade = hitGrade;
+				tempColor = "#7ed653";
+				tempShadow = "0 0 0.5rem #7ed653";
+				break;
+			case "B":
+				tempGrade = hitGrade;
+				tempColor = "#53d4d6";
+				tempShadow = "0 0 0.5rem #53d4d6";
+				break;
+			case "C":
+				tempGrade = hitGrade;
+				tempColor = "#d6538e";
+				tempShadow = "0 0 0.5rem #d6538e";
+				break;
+			case "D":
+				tempGrade = hitGrade;
+				tempColor = "#f04848";
+				tempShadow = "0 0 0.5rem #f04848";
+				break;
+			default:
+				tempGrade = "SS";
+				tempColor = (hdfl ? "#ffffff" : "#d6c253");
+				tempShadow = (hdfl ? "0 0 0.5rem #ffffff" : "0 0 0.5rem #d6c253");;
+				break;
+			}
+		}
+		//Game State Check
+		if (menu.state == 2) {
+			//Main
+			mainContainer.style.opacity = "1";
+			// Rank
+			rankCheck();
+			setRankStyle(tempGrade, tempColor, tempShadow);
+			//Box
+			document.documentElement.style.setProperty('--width', ` 500px`);
+			// Line
+			document.documentElement.style.setProperty('--progress', ` ${(menu.bm.time.current / menu.bm.time.full * 100).toFixed(2)}%`);

```

Comment:
As stated in the [wiki](https://github.com/l3lackShark/gosumemory/wiki/JSON-values) change bm.time.full to bm.time.mp3